docs/ADRs/012-tad-tool.md

```diff
@@ -0,0 +1,42 @@
+# ADR-0012 Tool for Transparency of Algorithmic Decision making
+
+## Context
+
+The Dutch government demands that government organisations are transparent about the use of
+high impact algorithms. This is a requirement of
+the [Open Government Act](https://www.rijksoverheid.nl/onderwerpen/wet-open-overheid-woo).
+This transparency can start with registering an algorithm in the
+[Algoritmeregister](https://algoritmes.overheid.nl/).
+
+Transparency and validation can be achieved by doing assessments,
+like [IAMA](https://www.rijksoverheid.nl/documenten/rapporten/2021/02/25/impact-assessment-mensenrechten-en-algoritmes)
+for decision making and technical evaluations to give insight in the features and outcomes of an algorithm. These
+outcomes can be combined to create a report that can be shared.
+
+There are several tools available on the market. Some support only assessments, others already
```

Comment:
```suggestion
There are several existing open source tools. Some support only assessments, others already
```

Moreover, maybe also include our shortlist so we can be as specific as possible.

Reply:
I have made the suggested change and added the link.